rollup.config.js

```diff
@@ -60,7 +69,7 @@ module.exports = [
       exports: 'named',
       sourcemap: true
     },
-    plugins,
+    plugins: plugins({ browser: false }),
```

Comment:
nit: since the default is `false` we can just do `plugins()` here.

Reply:
The way it was you had to actually call it with `plugins({})` to use the default value, which isn't very elegant either. So I changed the function signature so that it now does specify a default options object and it can be called with `plugins()` 😊